docs/testing/debug-office-add-ins-on-ipad-and-mac.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Debug Office Add-ins on a Mac
```

Comment:
Even though this article's title (this one and the H1) mentions Mac only, the article also talks about clearing the iPad cache. Should iPad be included in the title and intro text as appropriate?

Reply:
Good point, I noticed this as well. Given that the filename for this article contains "-on-ipad-and-mac", I suspect that at some point in the past this article did contain guidance for *both* iPad and Mac. However, since it currently only contains guidance for Mac, I think it makes sense to remove all mentions of iPad from the article.